src/Sdk/WebApi/WebApi/RawHttpClientBase.cs

```diff
@@ -68,6 +68,20 @@ public class RawHttpClientBase: IDisposable
             m_formatter = new VssJsonMediaTypeFormatter();
         }
 
+        protected RawHttpClientBase(
+            Uri baseUrl,
+            HttpMessageHandler pipeline,
+            string token,
+            bool disposeHandler)
+        {
+            m_client = new HttpClient(pipeline, disposeHandler);
+
+            // Disable their timeout since we handle it ourselves
+            m_client.Timeout = TimeSpan.FromMilliseconds(-1.0);
```

Comment:
Don't do this, play keep the timeout.
What if the client can't event reach your service, do we want it to sit there forever?

Reply:
I believe this is for a different purpose.  We will be uploading directly to Blob Storage, so maybe the intention was to allow an upload for longer?

However I agree we shouldn't change this timeout value.  I will check, but I don't believe today when uploading to Actions Service we have a different timeout value.

Reply:
Hmmm, I see this was already there in another block.  We don't need to introduce a new constructor so I removed it.